apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/commons/LoadedFileFilter.java

```diff
@@ -61,12 +61,12 @@ public boolean matchesDateRange(DateRangeParam dateRangeParam) {
     if (upperBound != null) {
       switch (upperBound.getPrefix()) {
         case LESSTHAN:
-          if (upperBound.getValue().getTime() <= getFirstUpdated().getTime()) {
+          if (upperBound.getValue().toInstant().isBefore(getFirstUpdated())) {
             return false;
           }
           break;
         case LESSTHAN_OR_EQUALS:
-          if (upperBound.getValue().getTime() < getFirstUpdated().getTime()) {
+          if (!upperBound.getValue().toInstant().isAfter(getFirstUpdated())) {
```

Comment:
Yes. We have a few tests I believe that would fail, because when I accidentally called a different method for converting the instants and comparing, I was getting failures. The way less than or equals was in this line before was on shaky ground, but the way we have it now is consistent with another part of the codebase that involves greater/greater than or =, etc.